exporter/collector/config.go

```diff
@@ -59,6 +59,16 @@ type MetricConfig struct {
 	// Buffer size for the channel which asynchronously calls CreateMetricDescriptor. Default
 	// is 10.
 	CreateMetricDescriptorBufferSize int `mapstructure:"create_metric_descriptor_buffer_size"`
+
+	// If true, the exporter will copy OTel's service.name, service.namespace, and
+	// service.instance.id resource attributes into the GCM timeseries metric labels. This
+	// option is recommended to avoid writing duplicate timeseries against the same monitored
+	// resource. Default is true.
```

Comment:
I feel like this behavior should be off by default.  It may make sense for this to be enabled by default for some ways of running the collector (i.e. maybe the ops-agent should enable this by default), but for OSS users, this behavior would be surprising to me.

Also, if we someday want to make the behavior better (e.g. confluence for resource attributes), writing these as metric labels by default might be confusing (it would show up as both a metric label and a resource attribute).

Reply:
See comment above about uniqueness. For OSS users, I think this should be on by default so metrics just work: 

- On `gce_instance`, you need some labels to differentiate different processes or you'll get the duplicate time series error
- In Python, Ruby, and other GIL'ed languages, it's common to use multiple processes on a single host for parallelism instead of threads. Even in a single container, it's common to do this

Reply:
@dashpole I think we are on the same page with this now. If so, can you resolve?